webapp-django/crashstats/base/jinja2/crashstats_base.html

```diff
@@ -143,6 +143,7 @@
 
     {% for status in status_messages %}
     <div class="status-message severity-{{ status.severity }}">
+        <span>(<time datetime="{{ status.date.isoformat() }}" class="ago">{{ status.date.strftime('%a, %b %d %H:%M %Z') }}</time>)</span>
```

Comment:
If so, do we need to rebase this PR? 
https://github.com/mozilla/socorro/pull/3420/files#diff-10764596e4964bff66024ef30db82ac7R21